pyproject.toml

```diff
@@ -0,0 +1,138 @@
+[project]
+name = "inference-development"
+description = "With no prior knowledge of machine learning or device-specific deployment, you can deploy a computer vision model to a range of devices and environments using Roboflow Inference."
+authors = [{ name = "Roboflow", email = "help@roboflow.com" }]
+requires-python = ">=3.8"
+readme = "README.md"
+license = { text = "Apache-2.0" }
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+]
+dynamic = ["version"]
+
+# Core dependencies
+dependencies = [
+    "PyYAML~=6.0.0",
+    "accelerate>=0.25.0,<=0.32.1",
+    "transformers>=4.41.1",
+    "pandas>=2.0.0,<2.3.0",
+    "requests",
+    "opencv-python>=4.8.1.78,<=4.10.0.84",
+    "onnxruntime-gpu>=1.15.1,<1.20.0",
+    "xformers",
+    "backoff~=2.2.0",
+    "py-cpuinfo~=9.0.0",
+    "python-dotenv<=2.0.0",
+    "pytest",
+    "einops>=0.7.0,<=0.8.0",
+    "torchvision>=0.15.0",
+    "rf-clip==1.1",
+    "requests_toolbelt",
+    "numpy<=1.26.4",
+    "supervision>=0.25.1,<=0.30.0",
+    "nvidia-ml-py<13.0.0",
+    "pybase64~=1.0.0",
+    "pillow",
+    "aiohttp>=3.9.0,<=3.10.11",
+    "rich>=13.0.0,<13.10.0",
+    "httpx>=0.25.1,<0.28.0",
+    "torch>=2.0.1,<=2.4.0",
+]
+
+[project.optional-dependencies]
```

Comment:
are we duplicating these here and in the `requirements.*.txt` files?  Is there a way to make sure we can keep them in sync or maybe we wouldnt need the requirement files anymore?

Reply:
This removes the needs for requirements.txt. Didn't remove them yet since I'm not done with the migration of the dockerfiles.